protobuf/flow/execution/execution.proto

```diff
@@ -19,15 +19,17 @@ service ExecutionAPI {
     // Scripts
 
     // ExecuteScriptAtBlockID executes a ready-only Cadence script against the execution state at the block with the given ID.
-    rpc ExecuteScriptAtBlockID (ExecuteScriptAtBlockIDRequest) returns (ExecuteScriptResponse);
+    rpc ExecuteScriptAtBlockID (ExecuteScriptAtBlockIDRequest) returns (ExecuteScriptAtBlockIDResponse);
 
     // Events
 
     // GetEventsForBlockIDs retrieves events for all the specified block IDs that have the given type
-    rpc GetEventsForBlockIDs (GetEventsForBlockIDsRequest) returns (EventsResponse);
+    rpc GetEventsForBlockIDs (GetEventsForBlockIDsRequest) returns (GetEventsForBlockIDsResponse);
```

Comment:
With the rename of `GetEventsForBlockIDTransactionID`, there only remains one Event related API. Renaming the response to be more specific